views/layouts/main.handlebars

```diff
@@ -16,17 +16,17 @@
   </head>
 
   <body>
-    <header class='buddy-card'>
-      <nav class='container text-center'>
-        <h1>Studdy Buddy</h1>
-        <section id='nav-btn' class='container'>
-          <a class='btn btn-grad' href='/'>Home</a>
+    <header>
+      <nav class='nav'>
+        <img id='nav-logo' src='nav-logo.png' />
```

Comment:
same goes here: this src wont work. has to be relative and pointing at image folder so something like ./images/file